packages/dom/src/dom/is-form-element.js

```diff
@@ -7,14 +7,28 @@ import isInputOrTextArea from './is-input-or-text-area';
  *
  * Detects if element is a form element.
  *
- * @param {Element} element The element to check.
+ * @param {Element} element                The element to check.
+ * @param {boolean} includeContentEditable Should include elements with contentEditable attribute.
  *
  * @return {boolean} True if form element and false otherwise.
  */
-export default function isFormElement( element ) {
+export default function isFormElement(
+	element,
+	includeContentEditable = false
```

Comment:
I wonder if instead of adding an extra parameter, another option would be adding an `isContentEditable` function to the dom package and using that instead.

I'm never a huge fan of boolean parameters, as the calling code is very unclear. For example `isFormElement( element, true )`, I have to look up what the `true` means.

An alternative is to add an options object, so it becomes `isFormElement( element, { includeContentEditable: true } );`, which is more verbose, but means readers don't have to look up documentation.

But I also think `isFormElement( element ) || isContentEditable( element )` is very readable and the separate function might be more useful in a wider range of situations.

PS: I think this function is unusual anyway. It doesn't return `true` for a `<form>` element, which contradicts the name 😆

Reply:
Removed this for now.